src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -369,14 +392,14 @@ private static IMetricsExporter getMetricsExporter(
     /**
      * This provides a singleton of the elasticSearch client that can be used by Guice.
      *
-     * The client is threadsafe so we don't need to keep creating new ones.
+     * The client is threadsafe, so we don't need to keep creating new ones.
      *
      * @param clusterName
      *            - The name of the cluster to create.
      * @param address
      *            - address of the cluster to create.
      * @param port
-     *            - port of the custer to create.
+     *            - port of the cluster to create.
```

Comment:
😹

Reply:
IntelliJ was warning me about this typo and I couldn't ignore it :P